cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1242,6 +1244,24 @@ private MapAndStatus<ConfigMap, KafkaRebalanceStatus> handleRebalanceResponse(Re
         }
     }
 
+    private static <T> Future<T> fromCompletableFuture(CompletableFuture<T> completableFuture) {
```

Comment:
I think this will be needed in other places as well as we move away from Vert.x. So it might be good to move this to `VertxUtil`? It already has similar method for Kafka Future. So you can put it next t it and name it something like `completableFutureToVertFuture`?

Reply:
that's a good idea. I will move this method to the common class to be reused.